File: fern/snippets/tool-calling/typescript/file-download.ts
import { Composio } from '@composio/core';
const composio = new Composio({apiKey: 'your-composio-api-key'});

// Download a file from Google Drive
const result = await composio.tools.execute('GOOGLEDRIVE_DOWNLOAD_FILE', {
    userId: 'user-1235***',
    arguments: {
      file_id: 'your-file-id'
    }
  });
  
// Result includes local file path
console.log(result);

File: fern/snippets/tool-calling/python/file-download.py
from composio import Composio


composio = Composio(
    api_key="your_composio_key", file_download_dir="./downloads"
)  # Optional: Specify the directory to download files to

result = composio.tools.execute(
    "GOOGLEDRIVE_DOWNLOAD_FILE",
    user_id="user-1235***",
    arguments={"file_id": "your_file_id"},
)

# Result includes local file path
print(result)
